CircleCI test_py job failing with Yarn install error

## Description
CircleCI `test_py` builds are failing installing Yarn. Seems to be happening on multiple [pull requests](https://app.circleci.com/pipelines/github/mozilla/bedrock/2571/workflows/224a78d3-f3de-45c9-a31d-79c13e3757d4/jobs/32984).

`EPERM: operation not permitted, copyfile '/usr/local/share/.cache/yarn/v6/npm-astral-regex-1.0.0-6c8c3fb827dd43ee3918f27b82782ab7658a6fd9-integrity/node_modules/astral-regex/index.js' -> '/app/node_modules/astral-regex/index.js'".
info If you think this is a bug, please open a bug report with the information provided in "/app/yarn-error.log`

```
Step 5/66 : RUN yarn install --pure-lockfile
 ---> Running in 272e65b2c5d2
yarn install v1.22.5
[1/4] Resolving packages...
[2/4] Fetching packages...
info fsevents@1.2.9: The platform "linux" is incompatible with this module.
info "fsevents@1.2.9" is an optional dependency and failed compatibility check. Excluding it from installation.
info fsevents@2.0.7: The platform "linux" is incompatible with this module.
info "fsevents@2.0.7" is an optional dependency and failed compatibility check. Excluding it from installation.
[3/4] Linking dependencies...
error An unexpected error occurred: "EPERM: operation not permitted, copyfile '/usr/local/share/.cache/yarn/v6/npm-astral-regex-1.0.0-6c8c3fb827dd43ee3918f27b82782ab7658a6fd9-integrity/node_modules/astral-regex/index.js' -> '/app/node_modules/astral-regex/index.js'".
info If you think this is a bug, please open a bug report with the information provided in "/app/yarn-error.log".
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
ERROR: Service 'assets' failed to build: The command '/bin/sh -c yarn install --pure-lockfile' returned a non-zero code: 1
Makefile:39: recipe for target 'build' failed
make[1]: *** [build] Error 1
make[1]: Leaving directory '/home/circleci/project'
Makefile:33: recipe for target '.docker-build' failed
make: *** [.docker-build] Error 2
```

--- a/.circleci/config.yml
+++ b/.circleci/config.yml
@@ -2,10 +2,11 @@ version: 2
 jobs:
   test_py:
     docker:
-      - image: circleci/python:2.7-stretch
+      - image: circleci/python:3.7-stretch
     steps:
       - checkout
-      - setup_remote_docker
+      - setup_remote_docker:
+          version: 19.03.13
       - run:
           name: Python Tests
           command: make clean test-image

